easybuild/easyconfigs/s/Salmon/Salmon-0.9.1-foss-2017a.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '0.9.1'
+
+homepage = 'https://github.com/COMBINE-lab/%(namelower)s'
+description = """Salmon is a wicked-fast program to produce a highly-accurate,
+ transcript-level quantification estimates from RNA-seq data."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = ['https://github.com/COMBINE-lab/%(namelower)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['3a32c28d217f8f0af411c77c04144b1fa4e6fd3c2f676661cc875123e4f53520']
+
+builddependencies = [('CMake', '3.8.1')]
+
+dependencies = [
+    # this is to align dependencies with BLAST+
+    ('Boost', '1.63.0', '-Python-2.7.13'),
```

Comment:
Please use `versionsuffix` here, like we do in existing `Salmon` easyconfigs (you'll need to rename the easyconfig file too):

```python
version = '0.9.1'
versionsuffix = '-Python-2.7.13'
...
dependencies = [
    # this is to align dependencies with BLAST+
    ('Boost', '1.63.0', versionsuffix),
...

Reply:
I'll go through the dependencies and match them up. For consistency: when I add an versionsuffix, that would also have to show up in the .eb filename?

In that specific case, I was using the Boost+Python version, to have the same dependency as the BLAST+ dependency. 
Or should I better use BLAST+ without Python?

Reply:
Only use Boost & BLAST+ with Python in versionsuffix if the Python bindings are required.